elide-swagger/src/main/java/com/yahoo/elide/swagger/resources/ApiDocsEndpoint.java

```diff
@@ -119,6 +119,15 @@ public Response listJson(
         }
     }
 
+    @GET
+    @Produces(MediaType.APPLICATION_JSON)
```

Comment:
Will these ever get invoked given that the other method matches all paths (@Path("{path:.*}")) including the empty path?

Reply:
Yes. From what I remember in certain JAX-RS setups for `@Path("{path:.*}")` where the path is empty, this will only match if there is a trailing `/`. So this will get invoked for cases where the path is empty and there is no trailing `/`.